handler/pkce/handler.go

```diff
@@ -116,12 +117,12 @@ func (c *Handler) validate(challenge, method string, client fosite.Client) error
 	case "":
 		if !c.EnablePlainChallengeMethod {
 			return errorsx.WithStack(fosite.ErrInvalidRequest.
-				WithHint("Clients must use code_challenge_method=S256, plain is not allowed.").
+				WithHintID(i18n.ErrHintInvalidPKCECodeChallengeMethod, "S256").
 				WithDebug("The server is configured in a way that enforces PKCE S256 as challenge method for clients."))
 		}
 	default:
 		return errorsx.WithStack(fosite.ErrInvalidRequest.
-			WithHint("The code_challenge_method is not supported, use S256 instead."))
+			WithHintID(i18n.ErrHintInvalidPKCECodeChallengeMethod, "S256"))
```

Comment:
This is no longer `WithHint("The code_challenge_method is not supported, use S256 instead."))`

Reply:
Addressed